gopherage/pkg/cov/junit/calculation/calculation_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package calculation
+
+import (
+	"testing"
+
+	"golang.org/x/tools/cover"
+)
+
+func TestCovList(t *testing.T) {
+	profiles := []*cover.Profile{
+		{FileName: "a", Mode: "count", Blocks: []cover.ProfileBlock{
```

Comment:
nit: is `gofmt` really ok with this? These keys should really be on their own lines so we don't get mixed levels of nesting at the same indentation like the bit below:
```
},
},
```
:grimacing:

Reply:
yes, gofmt is ok with this